CHANGES.rst

```diff
@@ -5368,7 +5368,7 @@ Incompatible changes
   use ``px`` rather than ignore it.
 * latex: Separate stylesheets of pygments to independent .sty file
 * #2454: The filename of sourcelink is now changed.  The value of
-  `html_sourcelink_suffix` will be appended to the original filename (like
+  ``html_sourcelink_suffix`` will be appended to the original filename (like
```

Comment:
I'll comment once but if you can use `confval` role when possible (or any other referencable role), I'd be happy (only on the text that needs to be changed).